spec/dockerfile_spec.rb

```diff
@@ -0,0 +1,53 @@
+#! /usr/bin/env ruby
+
+require 'rspec/core'
+require 'json'
+require "#{File.join(File.dirname(__FILE__), 'examples', 'running_cluster.rb')}"
+
+describe 'The docker-compose file works' do
+  def which(cmd)
```

Comment:
Did you have a problem running these commands on Windows? `PATHEXT` should come into play automatically, and you shouldn't need to resolve a command like this for any directory that is already part of `PATH`.